content/en/docs/Installation/_index.md

```diff
@@ -6,7 +6,7 @@ description: >
   Installation and configuration details for Kyverno using `Helm` or `kubectl`.
 ---
 
-Kyverno can be installed using Helm or deploying from the YAML manifests directly. When using either of these methods, there are no other steps required to get Kyverno up and running.
+Kyverno can be installed using Helm or deploying from the YAML manifests directly. When using either of these methods, there are no other steps required to get Kyverno up and running. Kyverno must always be installed in a dedicated Namespace; it must not be co-located with other applications in existing Namespaces including system-level Namespaces such as `kube-system`.
```

Comment:
```suggestion
Kyverno can be installed using Helm or deploying from the YAML manifests directly. When using either of these methods, there are no other steps required to get Kyverno up and running.
Kyverno must always be installed in a dedicated Namespace; it must not be co-located with other applications in existing Namespaces including system-level Namespaces such as `kube-system`.
```

Let's use some extra newlines. They will render as a single line but it makes editing and diffs more readable.

Reply:
Done.